### DISCUSSION TITLE
Do you have plan to improve Documentation?

### DISCUSSION THREAD

**Discussion by allestaire** [prisma/prisma#13046] [Q&A] [answered]
Seems like the documentation on the site is not on the go, you have to find it manually especially the list of model methods such are `findMany`,

Im not a **GraphQL** user, but I believe most of the developers are eager to use this package when the documentation is on-the-go.

It takes time to read all the documentation first and find the specific docs..
Laravel does this, as for example, hope you find way to improve the documentation

**Comment by nurul3101** [comment]
Hey @allestaire 👋 

When you say you have to "find it manually", do you mean that the search functionality in docs isn't working as expected for you?

**Comment by allestaire** [comment, marked as answer]
Thanks for the quick response @nurul3101 😄 

I dont seem like the search is helpful on my part,
But I guess it will be helpful if I know the keyworkds that I need to search
Im not yet fully experienced with Prisma, I only do quick search
Sorry for making this an issue.

But I guess, I have to understand the documentation first, it might not be the same as the others but as what I observe it is being design that way because of its different structure unlike others.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
